book/src/csi-driver-object.md

```diff
@@ -45,6 +48,12 @@ There are three important fields:
     * `"csi.storage.k8s.io/pod.namespace": pod.Namespace`
     * `"csi.storage.k8s.io/pod.uid": string(pod.UID)`
   * For more information see [Pod Info on Mount](pod-info.md).
+* `volumeLifecycleModes`
+  * This field was added in Kubernetes 1.16 and must not be set when using an older Kubernetes release.
+  * It informs Kubernetes about the volume modes that are supported by the driver.
```

Comment:
What does Kubernetes do with that information?

Reply:
Added: "This ensures that the driver is not used incorrectly by Kubernetes."

Reply:
Can you be more specific about what enforcement or behavior changes result from setting the fields? Or link to the inline page that describes that?